packages/service-clients/azure-client/package.json

```diff
@@ -109,7 +109,7 @@
 		"@biomejs/biome": "^1.7.3",
 		"@fluid-tools/build-cli": "^0.40.0",
 		"@fluidframework/aqueduct": "workspace:~",
-		"@fluidframework/azure-client-previous": "npm:@fluidframework/azure-client@2.0.0-rc.5.0.0",
+		"@fluidframework/azure-client-previous": "npm:@fluidframework/azure-client@2.0.0",
```

Comment:
Noting for follow up - find the commit which would have added the "broken" entries.